keymirror/keymirror.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for keymirror
+// Project: https://github.com/STRML/keyMirror
+// Definitions by: Johannes Fahrenkrug <https://github.com/jfahrenkrug>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'keymirror' {
```

Comment:
Write the declaration as a module. You can just remove this wrapper and it will still work, after this file is renamed to `index.d.ts`.